### DISCUSSION TITLE
Unable to republish the page | No "Your site is live at"

### DISCUSSION THREAD

**Discussion by PaperspaceMagazine** [community/community#67748] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I had a website running and was changing the DNS nameservers and the website went down. 

As a little test, I tried unpublishing the website in the pages section where it states "Your site is live at."

![image](https://github.com/community/community/assets/124681815/e8f43cdb-461e-435a-94a8-2d8cfc17413e)

After unpublishing, I see no statement to say that the website is live, only the above statement is no longer there. How do I republish? 

I can still see the following: 
![image](https://github.com/community/community/assets/124681815/3f129c28-2752-4c1d-849a-1cb628f3c189)

Apolgise if this is simple user error and thanks in advance.

**Comment by PaperspaceMagazine** [comment, marked as answer]
Solved! After clearing cache and waiting around 20 minutes, this problem went away and the site now shows to be published.

**Reply by queenofcorgis** [reply]
🎉 SLAY! Thanks for coming back and letting us know you got it solved ✅
